pkg/pki/privatekey.go

```diff
@@ -43,7 +45,17 @@ func ParsePEMPrivateKey(data []byte) (*PrivateKey, error) {
 }
 
 func GeneratePrivateKey() (*PrivateKey, error) {
-	rsaKey, err := rsa.GenerateKey(crypto_rand.Reader, 2048)
+	var rsaKeySize int64 = 2048
+
+	if os.Getenv("RSA_PRIVATE_KEY_SIZE") != "" {
+		var intErr error
+		rsaKeySize, intErr = strconv.ParseInt(os.Getenv("RSA_PRIVATE_KEY_SIZE"), 0, 0)
```

Comment:
Can we name it `KOPS_RSA_PRIVATE_KEY_SIZE` just to avoid possible collisions - I _think_ we've tried to prefix all our env vars like that just to keep them clear as well?

But then LGTM - and the `strconv.ParseInt(s, 0, 0)` trick was new to me (I've always used `strconv.Atoi`) but that's pretty cool :-)